components/clientComponents/forms/Review/Review.tsx

```diff
@@ -134,6 +186,16 @@ export const Review = ({ language }: { language: Language }): React.ReactElement
   // Focus heading on load
   useFocusIt({ elRef: groupsHeadingRef });
 
+  //Prior to useMemo, we grab the necessary translations for AddressComplete
+  addressCompleteStrings.unitNumber = t("addressComponents.unitNumber", { lng: language });
+  addressCompleteStrings.civicNumber = t("addressComponents.civicNumber", { lng: language });
+  addressCompleteStrings.streetName = t("addressComponents.streetName", { lng: language });
+  addressCompleteStrings.city = t("addressComponents.city", { lng: language });
+  addressCompleteStrings.province = t("addressComponents.province", { lng: language });
+  addressCompleteStrings.postalCode = t("addressComponents.postalCode", { lng: language });
+  addressCompleteStrings.country = t("addressComponents.country", { lng: language });
```

Comment:
This pattern of passing in language strings is unfortunate. I originally wrapped the related code in a useMemo but I didn't really see any performance difference. We (team) may want to think about a pattern/general idea when it makes sense to use useMemo. Also the React compiler will soon automatically add useMemo's where it makes sense in code. Sorry for rambling in your PR, just something to think about in the future as a team.